src/components/jsonTransforms/usfmjsToSlateRules.js

```diff
@@ -118,24 +109,31 @@ export const slateRules = [
     pathRule(
         '.tag',
         d => ({
-            "object": "inline",
+            "object": "block",
             "type": d.match,
-            "data": {"source": d.context},
+            "data": tagData(d.match, d.context),
             "nodes": [
-                d.context.text ? inlineTextNode(d.context) : null,
-                d.context.content ? inlineContentNode(d.context) : null
+                d.context.text ? bareTextNode(d.context.text) : null,
+                d.context.content ? bareTextNode(d.context.content) : null,
             ]
-                .concat(d.context.children ? d.runner(d.context.children) : null)

```

Comment:
I don't know under what circumstances there will be children, but we don't want children besides a single bare text node.